packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -191,12 +191,18 @@ const forms = {
         return get(contentType, ['schema', 'collectionName'], '');
       });
 
+      const takenCollectionNames = isEditing ? collectionNames.filter((collectionName) => {
+        const currentPluralName = get(contentTypes, [ctUid, 'schema', 'pluralName'], '');
```

Comment:
We should also be checking collectionName too, not just pluralName

Reply:
but the validation is on the pluralName field